### ISSUE TITLE
Installation Issues

### ISSUE THREAD

**Issue by Alex-Bishka** [jliu/graph-normalizing-flows#2]
Hi there, I am having some issues installing the packages from the requirements file. I've tried to install directly from the requirements file, but the following packages seem to be causing trouble for me:
- dm-sonnet-1.34
- tensorflow-plot
- graph_nets
- sklearn

Specifically the graph_nets package. I've also tried installing graph nets on its own from the graph nets repo, instead of via the requirements file. I've also tried creating a conda env in python3.5, but pretty much everything I have tried has caused problems for me (typically dependency issues using numpy, tensorflow, tensorboard, etc.). Any help resolving the setup of this repo would be greatly appreciated.

**Comment by Alex-Bishka** [comment]
Figured it out. Ubuntu 20.04 is not compatible with some of the libraries in use.
